### PR TITLE
Swagger alr

### PR DESCRIPTION
**Pull request by reid-andrew**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features
### Why is this change required? What problem does it solve?
- Adds SwaggerUI documentation of API endpoints
### Were there any challenges that arose while implementing this feature? If so, how were they addressed?
- Bit of a learning curve with swagger. 
- Converted from original JSON file to YML for readability & easier editing.

### CODE REVIEW

File: application/static/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Item'
+        '400':
+          description: Bad Request
+  /items/<id>:
+    get:
+      summary: Returns a selected items.
+      tags:
```
Comment (StephanieFriend) [inline application/static/swagger.yml:60]: Forgive me, but I'm going to be super picky. Is there anyway to update this to singular 'item'? So it reads 'Returns a selected item.'

File: application/static/swagger.yml [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Item'
+
+        '404':
+          description: Item not found
+    put:
+      summary: Updates a selected items.
+      tags:
```
Comment (StephanieFriend) [inline application/static/swagger.yml:73]: Same with this phrase

File: application/__init__.py [hunk 3]
```diff
@@ -2,13 +2,26 @@
...
+    API_URL = '/static/swagger.yml'
+    SWAGGERUI_BLUEPRINT = get_swaggerui_blueprint(
+        SWAGGER_URL,
+        API_URL,
+        config={
+            'app_name': "auctionable_change_api"
+        }
+    )
+    app.register_blueprint(SWAGGERUI_BLUEPRINT, url_prefix=SWAGGER_URL)
+
```
Comment (StephanieFriend) [inline application/__init__.py:24]: Looks clean and readable, great job!

**Review by StephanieFriend — APPROVED** [review]
This all looks great! Just some grammar suggestions which won't give any deployment issues. Can change or discuss at any time. :)

File: application/static/swagger.yml (hunk 1 above)
Comment (reid-andrew) [reply application/static/swagger.yml:60]: Good catch. Sloppy copy/paste job on my part.

**Review by kmcgrevey — APPROVED** [review]
Typo issues updated and verified.

**Pull request merged**